db/migrate/20180622140220_add_recurring_logic_to_sync_plan.rb

```diff
@@ -0,0 +1,11 @@
+class AddRecurringLogicToSyncPlan < ActiveRecord::Migration[5.1]
+  def change
+    Katello::SyncPlan.find_each do |sync_plan|
+      sync_plan.associate_recurring_logic
+      sync_plan.start_recurring_logic
+      sync_plan.foreman_tasks_recurring_logic.enabled = sync_plan.enabled
+    end
```

Comment:
This find_each block needs to go after the add_column below, because its setting a value on that column :)